ios/Podfile

```diff
@@ -1,9 +1,26 @@
-require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
-require_relative '../node_modules/react-native-permissions/scripts/setup'
+def node_require(script)
+  # Resolve script with node to allow for hoisting
+  require Pod::Executable.execute_command('node', ['-p',
+    "require.resolve(
+      '#{script}',
+      {paths: [process.argv[1]]},
+    )", __dir__]).strip
+end
+
+# Use it to require both react-native's and this package's scripts:
+node_require('react-native/scripts/react_native_pods.rb')
+node_require('react-native-permissions/scripts/setup.rb')
 
-platform :ios, '12.4'
-install! 'cocoapods', :deterministic_uuids => false
+platform :ios, min_ios_version_supported
```

Comment:
From where we take `min_ios_version_supported`? How can we control it?

Reply:
the variable is handled by helpers file of the react-native packages, if we need to raise up we can remove the variable and define it by ourself, ATM on 0.72 the min version is 12.4 on 0.73 would be 13.4 

source https://github.com/facebook/react-native/issues/41053#issuecomment-1770292532

Reply:
good idea, totally agree!

Reply:
Addressed in [1bd954c](https://github.com/pagopa/io-app/pull/5864/commits/1bd954cd67a7a0d4149ac9db03391b6b2f4edd62)